applications/EverlastingCash.md

```diff
@@ -0,0 +1,225 @@
+
+Based on the Polkadot/Kusama ecosystem, Everlasting Cash will be playing a huge role in the future DeFi market, thanks to:
+1. The adoption a crypto-collateralized mechanism to ensure the basic value of ELC. 
+2. The utilization of a reserve-based liquidity mining mechanism to issue additional ELC, so that ELC grows in an orderly manner with the expansion of demand.
+3. The anti-inflation model, and using the anti-inflation factor K to adjust the ELCaim (the pegged rate). The annual appreciation rate of ELC is roughly the same as the inflation rate of USD.
+4. The buffer mechanism with reserves for price falls, which avoid the death loop trap of algorithmic stablecoins. 
+  
+The ELC reserve pool will prioritize the tokens from the Polkadot system to build a solid stablecoin ecosystem.
+
+
+## Team :busts_in_silhouette:
+The team beyond Everlasting Cash is the Cycan team, an early-stage core pool of talents from around the world, with a background in finance, blockchain development and large-scale software architectures serving multiple industries, business development and media. 
+Cycan has a core team of developers numbering 10, including a development leader, 7 Rust developers, 1 product manager, and 1 front-end engineer.
+
+### Team members
+Cycan is directed by the following key roles:
+* Donald Gao: founder, financial model designer
+* Michele Ruberl, global tech partner, IT architect 
+* Vivi Lin, global partner, CMO 
+
+### Contact
+* **Contact Name:** Donald Gao
+* **Contact Email:** Donald@cycan.network
+* Website http://cycan.network
+
+### Legal Structure 
+* **Registered Address:** to be registered.
+* **Registered Legal Entity:** Cycan Foundation is being registered in Singapore, ETA March 2021.
+
+
+### Team's experience
+Cycan’s team has a long experience in the blockchain technology development and financial model designs.
+
+The team has developed public chains similar to Ethereum, cross-chain bridges connecting DPOS-based side chain and the notary network, and the open consortium chain with the rPBFT consensus mechanism. The team also has rich experience in smart contract development, including the Bancor-based decentralized transaction protocol, decentralized identity, distributed domain registration and auction, decentralized stable coin protocols, etc.
+
+Our developers are proficient in Rust, Java, C++, Solidity, Go and other development languages, and familiar with the Substrate development framework.
+
+The team also has extensive experiences in financial model designs, especially for the trust instruments and relevant model designs.
+
+Lastly, the team has a consolidated architectural and devops experience both in the context of experimental and in real-world large-scale deployments.
+
+
+### Team Code Repos
+* ELP-Runtime-node: https://github.com/CycanTech/ELP-Runtime-node 
+* ELC-contracts: https://github.com/CycanTech/ELC 
+
+### Team LinkedIn Profiles
+* Provided in private through Google Form.
```

Comment:
Which Google Form are you referring to? We don't have one for the Open Grants Program, and I also can't find a submission with your details in the General Grants one. If possible integrate your profiles here for transparency.

Reply:
> 您指的是哪种Google表单？我们没有开放式赠款计划的赠款，我也无法在“普通赠款”中找到包含您的详细信息的提交。如果可能，请在此处集成您的配置文件以提高透明度。

*OK, if necessary, we will directly write the Linkedin address of the core team members in the material. We will complete this update soon, thank you.

Reply:
> Which Google Form are you referring to? We don't have one for the Open Grants Program, and I also can't find a submission with your details in the General Grants one. If possible integrate your profiles here for transparency.

OK, we will directly write the Linkedin addresses of the core team members in the application, thank you.